test/gets.js

```diff
@@ -17,47 +17,46 @@ test('gets', function() {
          TypeError,
          'Invalid value\n' +
          '\n' +
-         'gets :: Accessible a => TypeRep -> Array String -> a -> Maybe b\n' +
-         '                        ^^^^^^^\n' +
+         'gets :: Accessible a => Function -> Array String -> a -> Maybe c\n' +
+         '                        ^^^^^^^^\n' +
          '                           1\n' +
          '\n' +
          '1)  [1, 2, 3] :: Array Number, Array FiniteNumber, Array NonZeroFiniteNumber, Array Integer, Array ValidNumber\n' +
          '\n' +
-         'The value at position 1 is not a member of ‘TypeRep’.\n');
+         'The value at position 1 is not a member of ‘Function’.\n');
 
-  throws(function() { S.gets(Number, null); },
+  throws(function() { S.gets(S.K(true), null); },
```

Comment:
Same for the `get` tests? Should I change all `S.K(Bool)` functions to appropriate `S.is(TypeRep)` functions?